components/iotserver-mgt/org.wso2.carbon.device.mgt.iot.common/src/main/java/org/wso2/carbon/device/mgt/iot/common/transport/mqtt/MqttConfig.java

```diff
@@ -0,0 +1,55 @@
+package org.wso2.carbon.device.mgt.iot.common.transport.mqtt;
+
+import org.wso2.carbon.device.mgt.iot.common.config.server.DeviceCloudConfigManager;
+import org.wso2.carbon.device.mgt.iot.common.config.server.datasource.ControlQueue;
+
+public class MqttConfig {
+	private String mqttQueueEndpoint;
+	private String mqttQueueUsername;
+	private String mqttQueuePassword;
+	private boolean isEnabled;
+
+	private static final String MQTT_QUEUE_CONFIG_NAME = "MQTT";
```

Comment:
better to keep class level variables together just before the instant variables for improved readbility